tests/api-compatibility/acceptable-breakages-v10.0.99.txt

```diff
@@ -0,0 +1,5 @@
+Compat issues with assembly Mono.Android:
+CannotAddAbstractMembers : Member 'Android.Telephony.CellInfo.CellIdentity' is abstract in the implementation but is missing in the contract.
```

Comment:
This introduces a semantic question:

The *intention* "going forward" -- API-R and beyond -- is that we will not have any more API breakages.  There will *just* be `v10.0`.

As such, we (presumably?) do *not* want to update `acceptable-breakages-*.txt`, because they aren't acceptable. :-)

That said, in order *actually* not have any API breakages, we will need `generator` fixes: https://github.com/xamarin/java.interop/issues/509

My concern is that if we merge this PR as-is, we'll forget to remove this `acceptable-breakages-v10.0.99.txt` file, and then miss any "actual" breakages when API-R goes stable.

Thus, instead of adding this file, could we instead make the `<CheckApiCompatibility/>` task invocation within `Mono.Android.targets` *conditional* on `%(AndroidApiInfo.Stable)`==True?  That way *no* API compatibility checking will be performed for *unstable* API levels, reducing the likelihood of us accidentally accepting API breakage in stable versions.

*Alternatively*, could we have a "different set" of `acceptable-breakages-*.txt` files to track unstable API levels?  `unstable-acceptable-breakages-*.txt`, perhaps?

Reply:
@gugavaro notes that the version number `v10.0.99` in this file should prevent us from accidentally including the contents into a stable version, so I think I'm overthinking things here.